docs/modules/usage/cloud/cloud-api.md

```diff
@@ -0,0 +1,168 @@
+# OpenHands Cloud API
+
+OpenHands Cloud provides a REST API that allows you to programmatically interact with the service. This is useful if you easily want to kick off your own jobs from your programs in a flexible way.
+
+This guide explains how to obtain an API key and use the API to start conversations.
+For more detailed information about the API, refer to the [OpenHands API Reference](https://docs.all-hands.dev/swagger-ui/).
+
+## Obtaining an API Key
+
+To use the OpenHands Cloud API, you'll need to generate an API key:
+
+1. Log in to your [OpenHands Cloud](https://app.all-hands.dev) account
+2. Navigate to the [Settings page](https://app.all-hands.dev/settings) by clicking on your profile icon in the bottom-left corner
+3. Locate the "API Keys" section
+4. Click "Generate New Key"
+5. Give your key a descriptive name (e.g., "Development", "Production")
+6. Copy the generated API key and store it securely - it will only be shown once
+
+![API Key Generation](/img/docs/api-key-generation.png)
+
+## API Usage
+
+### Starting a New Conversation
+
+To start a new conversation with OpenHands performing a task, you'll need to make a POST request to the conversation endpoint.
+
+#### Request Parameters
+
+| Parameter | Type | Required | Description |
+|-----------|------|----------|-------------|
+| `initial_user_msg` | string | Yes | The initial message to start the conversation |
+| `repository` | string | No | GitHub repository URL to provide context (must be a repository you've granted access to) |
```

Comment:
We should give an example for the repo param perhaps. This has to be `owner/repo` format, rather than URL links to the repo